episodes/18-preparing.md

```diff
@@ -48,27 +47,57 @@ but reduces a lot the preparation time of future lessons and are super useful to
+
+Create `setup.md` file in your lesson project root with a list of software and tools needed by participants to follow your lesson and links on how to install them.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+Another standard piece of The Carpentries Workbench are *Instructor Notes* pages - intended to help other instructors deliver your lesson by providing useful tips for delivering or explainng "sticky" points in the lesson. Instructor Notes for your lesson are located in `instructors/instructor-notes.md` file off your lesson project root where you may consider adding the following information:
+
+- rationale, strengths and/or weaknesses of the lesson design
+- what has been tried before and what worked/did not work
+- tips for teaching the lesson/particular sections
+- challenges encountered
+- troubleshooting common setup/installation/exercise issues 
+- parts that seem to cause trouble or most confusion for learners (i.e. where to tread carefully, spend more time explaining)
+    
+It may initially be difficult to create Instructor Notes until you have taught the lesson yourself for the first time - but after the first pilot you will have enough feedback to draft them.
+
+::::::::::::::::::::::::::::::::::::::  challenge
+
+## Exercise: Create Instructor Notes (10 minutes)
+
+Create `instructors/instructor-notes.md` file in your lesson project root with a list of delivery instructions for future instructors.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Feedback Collection Plan
+
+There is a lot to collect feedback on when trialling a new lesson and this is arguably the most important part of your teaching plan. It can be hard to capture all this information yourself while you are teaching, hence you will need help from your team - make sure you assign the feedback collection roles ahead of your pilot. Additionally, from the very beginning (i.e. advertising stage) make it clear to the participants you are piloting a new lesson and that they will be actively helping you refine it as well as learning new things themselves. This can help focus their attention on the feedback you need to collect from them, which you should be doing:
+
+- constantly and throughout the workshop - refer to the ["Pilot Workshops" section](17-operations.md#pilot-workshops) for suggestions on things to track and the [pilot workshop notes template](pilot-notes-template) and other suggested feedback collection mechanisms to help you gather such information. To recap, you should be noting down the amount of time used to teach each section, amount of time used for each exercise, technical issues encountered during the lesson, bugs and unexpected behaviour, missing solutions, questions and suggestions from participants, sources of confusion, interesting discussions, etc. These may occur at any point at the workshop, either verbally or in the chat function of the video conferencing tool (for online workshops) - make sure you capture them all.
```

Comment:
```suggestion
- __constantly and throughout the workshop__ - refer to the ["Pilot Workshops" section](17-operations.md#pilot-workshops) for suggestions on things to track and the [pilot workshop notes template][pilot-notes-template] and other suggested feedback collection mechanisms to help you gather such information. To recap, you should be noting down the amount of time used to teach each section, amount of time used for each exercise, technical issues encountered during the lesson, bugs and unexpected behaviour, missing solutions, questions and suggestions from participants, sources of confusion, interesting discussions, etc. These may occur at any point at the workshop, either verbally or in the chat function of the video conferencing tool (for online workshops) - plan to have someone try to capture as much as possible.
```
Think the bold might help make the first part stand out.
Also suggested less strong language at the end as it made me a bit stressed in its original form.